packages/react-ui-core/src/Gmap/Gmap.js

```diff
@@ -1,86 +1,162 @@
 import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
-import autobind from 'autobind-decorator'
-import { parseArgs } from '@rentpath/react-ui-utils'
-import GmapBase from './GmapBase'
+import themed from 'react-themed'
+import classnames from 'classnames'
+import startsWith from 'lodash/startsWith'
+import pickBy from 'lodash/pickBy'
+import { GmapInteraction } from './GmapInteraction'
+import withGoogleScript from './withGoogleScript'
 
-const API_BASE_URL = 'https://maps.googleapis.com/maps/api/js'
+const DEFAULT_ZOOM = 8
+const DEFAULT_CENTER = { // Atlanta, GA
+  lat: 33.7490,
+  lng: -84.3880,
+}
+
+const EVENTS = {
+  onBoundsChanged: 'bounds_changed',
+  onCenterChanged: 'center_changed',
+  onClick: 'click',
+  onDoubleClick: 'dblclick',
+  onDrag: 'drag',
+  onDragEnd: 'dragend',
+  onDragStart: 'dragstart',
+  onHeadingChnaged: 'heading_changed',
+  onIdle: 'idle',
+  onMaptypeIdChanged: 'maptypeid_changed',
+  onMouseMove: 'mousemove',
+  onMouseOut: 'mouseout',
+  onMouseOver: 'mouseover',
+  onProjectionChanged: 'projection_changed',
+  onResize: 'resize',
+  onRightclick: 'rightclick',
```

Comment:
`onRightClick` instead of `onRightclick`?

Reply:
Nice catch!